layouts/shortcodes/contribute.html

```diff
@@ -32,4 +59,132 @@
     
     <input class="db w-100 mv2 white pa3 bn hover-shadow hover-bg-black bg-animate bg-black" id="contributeButton" type="submit" value="Submit request" />
 
+    <div id="formResultsWrapper">
+        <em id="formResultsIntroValid">Pull request created:</em>
+        <em id="formResults"></em>
+    </div>
 </form>
+
+<script>
+    document.addEventListener("DOMContentLoaded", () => {
+        let form = document.querySelector("form");
+        let formResultsWrapper = document.querySelector("#formResultsWrapper");
+        let formResultsIntroValid = document.querySelector("#formResultsIntroValid");
+        let formResults = document.querySelector("#formResults");
+        
+        form.onsubmit = () => {
+            let formData = new FormData(form);
+            let formResultsLink = document.createElement("a");
+
+            // id to delete later from DOM
+            formResultsLink.setAttribute("id", "formResultsLink");
+            
+            // certain inputs should be parsed as an array
+            // store form data in an object as key-value pairs
+            let contributionObject = formToContribute(formData);
+
+            // trim values in the object
+            trimAll(contributionObject);
+
+            // JSON format the object
+            let jsonFormData = JSON.stringify(contributionObject);
+
+            // send JSON data as POST request in body using fetch API
+            const ENDPOINT = "https://us-central1-asianamericans-wiki.cloudfunctions.net/contribute-api";
+            let requestStatus = false;
+            fetch(ENDPOINT, {
+                method: "POST",
+                body: jsonFormData,
+            })
+                .then((response) => {
+                    // console.log("status: " + response.status);
+                    if (response.status === 201) {
+                        requestStatus = true;
+
+                        // accepted => green
+                        formResultsWrapper.style.background = "#2ECC40";
```

Comment:
create variables for the colors and remove these comments:
`// accepted => green`